### PR TITLE
Blog/maureen josephine/tech communities

### PR DESCRIPTION
**Pull request by JosephineAkello**
Create a Blog Post On Tech Communities and their links.

Check past Twitter post regarding tech communities that inspired this blog https://twitter.com/W_TechShoutOuts/status/1401222458892599298

### CODE REVIEW

File: content/posts/2021/01/2021-6-7-tech-communities/index.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+### 23. Product School
+
+Product School offers trainings to Product Enthusiasts and prepares students for product management interviews to enable them expand their Proffesssional Network through access to a Private Community of Product Experts. Find out more
+[<span style="color:blue"> **here**</span>](https://productschool.com/).
+
+### 24. DigiClan Africa
+
+This is a community of Digital Marketing Professionals and Enthusiasts who focus on training people to be better Content creators through Digital Marketing. Check it out [<span style="color:blue"> **here**</span>](https://www.digiclan.africa/).
+
+### 25. Reddit
```
Comment (TheBeachMaster) [inline content/posts/2021/01/2021-6-7-tech-communities/index.md]: Replace this with HackerNews [link](https://news.ycombinator.com)

Reddit is like a Social media app

**Review by TheBeachMaster — COMMENTED** [review]
See comments above

File: content/posts/2021/01/2021-6-7-tech-communities/index.md (hunk 1 above)
Comment (JosephineAkello) [reply content/posts/2021/01/2021-6-7-tech-communities/index.md]: @TheBeachMaster Ain't Hackernews also a social site as well?

Comment (TheBeachMaster) [reply content/posts/2021/01/2021-6-7-tech-communities/index.md]: It's a discussion board for techies... Reddit has a Twitter-like feel

**Review by TheBeachMaster — APPROVED** [review]

**Review by 75pollet — APPROVED** [review]

**Pull request merged**